cluster/juju/prereqs/ubuntu-juju.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2014 Canonical LTD. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# If you find any bugs within this script - please file bugs against the
+# Kubernetes Juju Charms project - located here: https://github.com/whitmo/bundle-kubernetes
+
+function check_for_ppa(){
+    grep -s ^ /etc/apt/sources.list /etc/apt/sources.list.d/* | grep juju
```

Comment:
Why not just:

```shell
  grep -qsw juju /etc/apt/sources.list /etc/apt/sources.list.d/*
```

Use `-q` to make it quiet and `-w` to make sure you're matching a whole word (i.e. you want to match `juju` but not `jujuba`).